pytensor/link/pytorch/linker.py

```diff
@@ -84,9 +86,16 @@ def fn(*inputs, inner_fn=inner_fn):
         return fn
 
     def create_thunk_inputs(self, storage_map):
+        from pytensor.link.pytorch.dispatch import pytorch_typify
```

Comment:
You'll need to copy the logic with SharedVariables in JAX to emmit a warning and use different variables. You can refactor the logic so it's not duplicated